tests/rest/RestCloudTest.php

```diff
@@ -16,11 +16,6 @@ final class RestCloudTest extends RestTestCase
   private const SchemaFilename20 = "/keymanweb-cloud-api/keymanweb-cloud-api-2.0.json";
   private const SchemaFilename10 = "/keymanweb-cloud-api/keymanweb-cloud-api-1.0.json";
 
-  static function setUpBeforeClass(): void
```

Comment:
This is not necessary; it is done in the parent class and the parent class also sets up the tier to 'test' correctly.